doc/environment_variables.md

```diff
@@ -0,0 +1,61 @@
+gRPC environment variables
+--------------------------
+
+gRPC C core based implementations (those contained in this repository) expose
+some configuration as environment variables that can be set.
+
+* GRPC_ABORT_ON_LEAKS
+  A debugging aid to cause a call to abort() when gRPC objects are leaked past
+  grpc_shutdown()
+
+* GRPC_GOOGLE_CREDENTIALS_ENV_VAR
+  The path to find the credentials to use when Google credentials are created
+
+* GRPC_SSL_CIPHER_SUITES
```

Comment:
could you add an example of how a cipher suite name looks like, or link to the OpenSSL reference for one?